internal/cmdrender/executor.go

```diff
@@ -82,15 +81,9 @@ func (e *Executor) Execute(ctx context.Context) error {
 		return err
 	}
 
-	// format resources before writing
-	_, err = filters.FormatFilter{UseSchema: true}.Filter(hctx.root.resources)
-	if err != nil {
-		return err
-	}
-
 	if e.Output == nil {
 		// the intent of the user is to modify resources in-place
-		pkgWriter := &kio.LocalPackageReadWriter{PackagePath: string(root.pkg.UniquePath)}
+		pkgWriter := &kio.LocalPackageReadWriter{PackagePath: string(root.pkg.UniquePath), PreserveSeqIndent: true}
```

Comment:
I didn't see the change for preserving SeqIndent where we read the resources. The method involved is `pkg.LocalResources()` or something.

Reply:
All reader logic calls are updated.

Reply:
Yes, I can see the one in `pkg/pkg.go`.